core/src/main/scala/org/apache/spark/scheduler/ReplayListenerBus.scala

```diff
@@ -107,11 +107,11 @@ private[spark] class ReplayListenerBus extends SparkListenerBus with Logging {
         }
       }
     } catch {
-      case ioe: IOException =>
-        throw ioe
-      case e: Exception =>
-        logError(s"Exception parsing Spark event log: $sourceName", e)
+      case ex: MalformedInputException =>
```

Comment:
@sharkdtu Can you please point out which code will throw this `MalformedInputException `?